aea/connections/p2p/stub/connection.yaml

```diff
@@ -0,0 +1,20 @@
+name: stub
+author: fetchai
+version: 0.1.0
+description: The stub p2p connection implements a local p2p connection allowing agents
+  to communicate with each other through files created in the namespace directory.
+license: Apache-2.0
+aea_version: '>=0.3.0, <0.4.0'
+# TOFIX(LR) which algo is used to compute fingerprints?
```

Comment:
you can use `aea fingerprint` You should also add the file to `scripts/generate_ipfs_hashes.py` and run that